### DISCUSSION TITLE
Please publish YouTube video guide😭🙏

### DISCUSSION THREAD

**Discussion by REFLEXGAMING007** [soranoo/TradingView-Free-Webhook-Alerts#30] [Q&A] [answered]
If you have little bit spare time. It would be really helpful if you publish a youtube video showing the full guide for local setup. I'm sure alot of other people want this too. For a coding noob like me its really hard.

**Comment by soranoo** [comment, marked as answer]
Let's check out https://www.youtube.com/playlist?list=PLOHaKcov3Nkt0LIK1joPYgFnZY24zf_Wo

Hope it can help.

**Reply by REFLEXGAMING007** [reply]
Thank you soo much pal💙
